vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -6,17 +6,20 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
+          "1.24.10-hotfix.20230509-2",
           "1.24.10-hotfix.20230509.1",
+          "1.24.9-hotfix.20230509-2",
           "1.24.9-hotfix.20230509.1",
+          "1.25.5-hotfix.20230509-3",
           "1.25.5-hotfix.20230509.1",
-          "1.25.5-hotfix.20230509.2",
```

Comment:
note: as reported by trivy, there is no different in terms of CVE between `1.25.5-hotfix.20230509.1` and `1.25.5-hotfix.20230509.2`, let's skip it and jump to `1.25.5-hotfix.20230509.3`